modules/recommended-scale-practices.adoc

```diff
@@ -0,0 +1,16 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/recommended-cluster-scaling-practices.adoc
+
+[id="recommended-scale-practices_{context}"]
+= Recommended practices for scaling the cluster
+
+When scaling up the cluster to higher node counts, you should spread the nodes
+across all of the available zones for higher availability. If the replicas in
+the machinesets are set to higher numbers all at one time, the controller might
+have issues creating the machines. The number of requests the cloud platform,
+which {product-title} is deployed on top of, is able to handle impacts the
+process. The controller will start to loop more while trying to create, check,
```

Comment:
@chaitanyaenr What do you mean that it loops more? I am wondering if we can use a more precise term that will translate better. Thanks!

Reply:
@ahardin-rh we can replace "loop" with "query".

Reply:
@chaitanyaenr Perfect. Thanks!